build/dependencies.props

```diff
@@ -76,20 +97,7 @@
     <MicrosoftExtensionsTypeNameHelperSourcesPackageVersion>2.1.1</MicrosoftExtensionsTypeNameHelperSourcesPackageVersion>
     <MicrosoftExtensionsValueStopwatchSourcesPackageVersion>2.1.1</MicrosoftExtensionsValueStopwatchSourcesPackageVersion>
     <MicrosoftExtensionsWebEncodersPackageVersion>2.1.1</MicrosoftExtensionsWebEncodersPackageVersion>
-    <MicrosoftNETCoreApp20PackageVersion>2.0.0</MicrosoftNETCoreApp20PackageVersion>
-    <MicrosoftNETCoreApp21PackageVersion>2.1.1</MicrosoftNETCoreApp21PackageVersion>
     <MicrosoftNetHttpHeadersPackageVersion>2.1.1</MicrosoftNetHttpHeadersPackageVersion>
     <MicrosoftNETSdkRazorPackageVersion>2.1.1</MicrosoftNETSdkRazorPackageVersion>
```

Comment:
Should the Sdk be unpinned? It's not referenced by runtime packages

Reply:
Is this a test-only dependency? If so, then yes, this can move up into the auto section.

Reply:
Yup - https://github.com/aspnet/Mvc/search?q=MicrosoftNETSdkRazorPackageVersion&unscoped_q=MicrosoftNETSdkRazorPackageVersion